staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset.go

```diff
@@ -662,6 +662,33 @@ func (qs *queueSet) cancelWait(req *request) {
 	qs.obsPair.RequestsWaiting.Add(-1)
 }
 
+// canAccommodateSeatsLocked returns true if this queueSet has enough
+// seats available to accommodate a request with the given number of seats,
+// otherwise it returns false.
```

Comment:
I think this PR should update this comment, perhaps to something like the following.

> // canAccommodateSeatsLocked decides whether the QueueSet's concurrency constraint allows dispatching a new request that will occupy the given number of seats.

Reply:
@tkashem - let's do that in a follow up